app/views/profiles/show.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class="row">
```

Comment:
This `div` isn't needed, since the view ends up inside the `row` defined in `application.html.erb`.